Controller/signup_screen.py

```diff
@@ -0,0 +1,87 @@
+"""_module summary_"""
+import importlib
+import time
+
+import multitasking
+
+import View.SignupScreen.signup_screen
+from View import SignupScreenView
+
+
+# We have to manually reload the view module in order to apply the
+# changes made to the code on a subsequent hot reload.
+# If you no longer need a hot reload, you can delete this instruction.
+importlib.reload(View.SignupScreen.signup_screen)
+
+
+class SignupScreenController:
+    """
+    The `MainScreenController` class represents a controller implementation.
+    Coordinates work of the view with the model.
+    The controller implements the strategy pattern. The controller connects to
+    the view to control its actions.
+    """
+
+    def __init__(self, model):
+        self.model = model  # Model.main_screen.MainScreenModel
+        self.views = [SignupScreenView(self, self.model)]
+
+    def clear_txt_field(self):
+        self.views[0].ids.user.text = ""
+        self.views[0].ids.passw.text = ""
+        self.views[0].ids.conpass.text = ""
+
+    def disabled_btn(self):
+        if self.views[0].ids.btn.disabled == False:
+            self.views[0].ids.btn.disabled =True
+    
+    def pass_data(self):
+        self.userdata = self.views[0].get_userdata()
```

Comment:
yung `self.userdata` ba is diyan lang yung scope? if yes, gawin mo nalang userdata